test/recipes/15-test_ml_dsa_codecs.t

```diff
@@ -86,7 +86,7 @@ foreach my $alg (@algs) {
         my $s = sprintf("sig-%s.%d.dat", $alg, $i++);
         ok(run(app([qw(openssl pkeyutl -sign -rawin -inkey), $sk, '-in', $der0,
                     qw(-pkeyopt deterministic:1 -out), $s])));
-        ok(!compare($s, $refsig),
+        ok(!compare_text($s, $refsig),
```

Comment:
This is again binary data, I do not think this is right.

Reply:
Fixed in a new commit.